server/pom.xml

```diff
@@ -336,16 +336,6 @@
       <artifactId>artemis-stomp-protocol</artifactId>
       <scope>compile</scope>
     </dependency>
-    <dependency>
-      <groupId>org.apache.qpid</groupId>
-      <artifactId>qpid-common</artifactId>
-      <scope>compile</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.qpid</groupId>
-      <artifactId>qpid-client</artifactId>
-      <scope>compile</scope>
-    </dependency>
     <dependency>
       <groupId>geronimo-spec</groupId>
```

Comment:
This dependency should have also been removed from here. Running `./gradlew pom` is removing it when I test your branch locally

Reply:
Removed ```geronimo-spec``` dependency.